### DISCUSSION TITLE
Beta 0.8 unexpected mjpeg stream for debugging output

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#351] [General]
I use Reolink IP cameras and it's working with 0.7.0 version without any issue, but with 0.8.0, it is showing following output. I am using the minimal config from readme and I have also tried adding the ffmpeg input flags that is working in 0.7 version but same result.

![image](https://user-images.githubusercontent.com/7528001/101794164-71f35580-3abb-11eb-8992-779f238df2a9.png)
![back](https://user-images.githubusercontent.com/7528001/101794262-8b949d00-3abb-11eb-9086-25061c408a78.jpg)

**Comment by blakeblackshear** [comment]
Can you post the config you used when adding the input flags?

**Reply by bm55** [reply]
![image](https://user-images.githubusercontent.com/7528001/101796052-66a12980-3abd-11eb-9524-047fb2049783.png)

**Reply by jamos77** [reply]
Have you tried swapping your height and width around - I got a similar feed when I had them the wrong way round.
height: 480
width: 640

**Reply by bm55** [reply]
Yes, that fixed it! Thanks.
Is height and width required or will it try to auto detect it if I remove it form config?

**Reply by blakeblackshear** [reply]
It is required in 0.8.0. The auto detection wasn't reliable enough.

**Comment by Trashmee** [comment]
I had trouble with my Reolink 511 (all green picture) - while various Foscams worked. For me increasing the fps from 5 to 10 seemed to make a difference - just FYI.
